modules/stanford_news_extras/stanford_news_extras.install

```diff
@@ -41,6 +41,13 @@ function stanford_news_extras_update_7301(&$sandbox) {
   stanford_news_extras_set_banner_image();
 }
 
+/**
+ * Implements hook_update_N().
+ */
+function stanford_news_extras_update_7301(&$sandbox) {
```

Comment:
```suggestion
function stanford_news_extras_update_7302(&$sandbox) {
```

Reply:
Yeah. I left that in because the docs say to keep updates for last two major revision. 

>A good rule of thumb is to remove updates older than two major releases of Drupal. 
https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7.x
Source:

Reply:
No, your new update hook needs to increment from `7301` to `7302`.

Reply:
```
spr-jbickar-9: ~$ drush @swssoe.dev sqlq --extra="-t" 'select name, schema_version from system where name="stanford_news_extras"'
+----------------------+----------------+
| name                 | schema_version |
+----------------------+----------------+
| stanford_news_extras |           7301 |
+----------------------+----------------+
```

Reply:
Oh weird. I did a search for an update with 7301 and couldn't locate one. 

Okay. Updating and pushing.